column.go

```diff
@@ -12,19 +12,19 @@ func (c columns) render() string {
 	rows := []string{}
 
 	for _, item := range c {
-		rows = append(rows, fmt.Sprintf("`%s` %s", item.field, item.definition.buildRow()))
+		rows = append(rows, fmt.Sprintf("`%s` %s", item.field, item.definition.BuildRow()))
 	}
 
 	return strings.Join(rows, ", ")
 }
 
 type column struct {
 	field      string
-	definition columnType
+	definition ColumnType
 }
 
-type columnType interface {
-	buildRow() string
+type ColumnType interface {
```

Comment:
Why did you change it? It works perfectly fine in private scope. Any reason it should be exported.

Also, it is out of the scope of the MR ❗

Reply:
If it is exported, developers can implement it more flexibly in outsourcing. For example, I found that when the blob type has different characteristics in different versions, it can solve similar problems externally, and the Customize queries part of the document cannot be implemented externally.